docs/step-2.md

```diff
@@ -62,8 +74,10 @@ curl -s -o /dev/null -w "Return Code: %{http_code}\n" <your-endpoint>
 
 
 
-Now we will customise our Scanner and Probe rules. These use Amazon Athena.
-The solutions refer to the Athena by a saved query ID. As Athena don't allow you to change saved queries, the process to apply customizations to Athena query is by creating a new query and updating the Athena log parser event to use the new query ID.
+Now you will customise our Scanner and Probe rules. These use Amazon Athena to query the logs generated by an application.
+The solutions refer to Athena by a saved query ID. As Athena don't allow you to change saved queries, the process to apply customizations to Athena query is by creating a new query and updating the Athena log parser event to use the new query ID.
```

Comment:
I was thinking of simplifying this a little bit, what do you think?

```suggestion
The solutions refer to Athena by a saved query ID. As Athena doesn't allow you to change saved queries, you need to create a new query and update the Athena log parser event to use the new query ID.
```

Reply:
Good observation. I've changed all the instances _iz_ to _is_ 😄